bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java

```diff
@@ -113,4 +129,43 @@ public ImageData getComponentData() {
         return super.getComponentData();
     }
 
+    @Override
+    public String getSrcset() {
+        String [] srcsetArray = new String[super.getWidths().length];
```

Comment:
I would cache `super.getWidths()` to a local variable, since it's used quite a lot.

Reply:
Cached it in https://github.com/adobe/aem-core-wcm-components/pull/1594/commits/34e98d07c503ba95ef5a84ec7ea590c981b3a535.